<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1086</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gas Analizer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/5980347_analysis_bar_chart_data_graph_icon.png</normaloff>icons/5980347_analysis_bar_chart_data_graph_icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="plainTextEdit">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>131</width>
      <height>521</height>
     </rect>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContents</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
    <property name="plainText">
     <string>측정 데이터가 표시됩니다.</string>
    </property>
    <property name="overwriteMode">
     <bool>true</bool>
    </property>
    <property name="maximumBlockCount">
     <number>0</number>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_measure">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_stop">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>61</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Stop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Measuring Data</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>30</y>
      <width>891</width>
      <height>591</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>891</width>
      <height>591</height>
     </size>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1086</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>파일</string>
    </property>
    <widget class="QMenu" name="menuSaveAS">
     <property name="title">
      <string>SaveAS</string>
     </property>
     <addaction name="actionSave_SQL"/>
     <addaction name="actionSave_Excel"/>
    </widget>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionClose"/>
    <addaction name="menuSaveAS"/>
   </widget>
   <widget class="QMenu" name="menu_measure">
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>측정</string>
    </property>
    <addaction name="actionMeasureStart"/>
    <addaction name="actionPause"/>
    <addaction name="actionRestart"/>
    <addaction name="actionMeasureStop"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu_setting">
    <property name="title">
     <string>설정</string>
    </property>
    <addaction name="actionCOM_Setting"/>
    <addaction name="actionEnvironment"/>
   </widget>
   <widget class="QMenu" name="menu_view">
    <property name="title">
     <string>보기</string>
    </property>
    <addaction name="actionWindow"/>
    <addaction name="actionCascade"/>
    <addaction name="actionZoom_In_Out"/>
    <addaction name="actionWindow_Arrange"/>
    <addaction name="actionMax"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_measure"/>
   <addaction name="menu_setting"/>
   <addaction name="menu_view"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionSave"/>
   <addaction name="separator"/>
   <addaction name="actionMeasureStart"/>
   <addaction name="actionMeasureStop"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionEnvironment">
   <property name="text">
    <string>Environment</string>
   </property>
  </action>
  <action name="actionWindow">
   <property name="text">
    <string>New Window</string>
   </property>
  </action>
  <action name="actionCascade">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionZoom_In_Out">
   <property name="text">
    <string>Zoom In / Out</string>
   </property>
  </action>
  <action name="actionWindow_Arrange">
   <property name="text">
    <string>Window Arrange</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="icon">
    <iconset>
     <normaloff>icons/1493292_new_document_page_sheet_icon.png</normaloff>icons/1493292_new_document_page_sheet_icon.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>icons/285657_floppy_guardar_save_icon.png</normaloff>icons/285657_floppy_guardar_save_icon.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionMeasureStart">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/22930_play_player_right_start_icon.png</normaloff>icons/22930_play_player_right_start_icon.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="shortcutContext">
    <enum>Qt::WidgetShortcut</enum>
   </property>
   <property name="menuRole">
    <enum>QAction::ApplicationSpecificRole</enum>
   </property>
  </action>
  <action name="actionMeasureStop">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icons/22947_red_stop_icon.png</normaloff>icons/22947_red_stop_icon.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionMax">
   <property name="text">
    <string>Max</string>
   </property>
  </action>
  <action name="actionCOM_Setting">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>COM Setting</string>
   </property>
  </action>
  <action name="actionSave_SQL">
   <property name="text">
    <string>Save_SQL</string>
   </property>
  </action>
  <action name="actionSave_Excel">
   <property name="text">
    <string>Save_Excel</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>btn_measure</tabstop>
  <tabstop>btn_stop</tabstop>
  <tabstop>plainTextEdit</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>actionMeasureStart</sender>
   <signal>triggered()</signal>
   <receiver>btn_measure</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>70</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionMeasureStop</sender>
   <signal>triggered()</signal>
   <receiver>btn_stop</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>140</x>
     <y>117</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
